gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
I'm not familiar with VS code's debugging, but I'd be surprised if generic gulp was supported and raw mocha wasn't. In any case, let's keep the main repo gulp-free to save on dependencies we don't strictly need 😉